src/main/java/CalcServlet.java

```diff
@@ -0,0 +1,49 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@WebServlet(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=5162544)